### ISSUE TITLE
0 results when radius < 1

### ISSUE THREAD

**Issue by wujekbogdan** [birgire/geo-query#6]
When `radius` is between `0` and `1` then the query doesn't give any results. Is it a bug, or is it a desired behavior?

**Comment by birgire** [comment]
Thanks @wujekbogdan,

Currently there's an integer parsing for the radius, with:

```
$groupby .= $this->db->prepare(
            " HAVING distance_value <= %d ",
            $this->radius
        );

```
but we can support floating point radius with: 

```
$groupby .= $this->db->prepare(
            " HAVING distance_value <= %f ",
            $this->radius
        );

```
I will update this in the next version, with corresponding integration tests.

**Comment by wujekbogdan** [comment]
@birgire 
It would be great. At the moment it's a bit misleading because `GeoQueryHaversine->setup` uses `floatval` to parse lat/lng/radius so it looks like floats are valid values not only for lat/lng but for the radius as well.

**Comment by birgire** [comment]
@wujekbogdan I agree, 

This is should be fixed in version 0.0.6

Thanks again.

**Comment by wujekbogdan** [comment]
@birgire 
Thanks a lot for a quick update!
